web/forms/implementing_simple_validation/my_element.dart

```diff
@@ -6,8 +6,8 @@ import 'package:polymer/polymer.dart';
 
 @PolymerRegister('my-element')
 class MyElement extends PolymerElement {
-  static const int _MIN_LENGTH = 5;
-  static const int _MAX_LENGTH = 50;
+  static const int _minLength = 5;
```

Comment:
haha this is a controversial one, but I think lining up with the style guide is probably best :)